### PR TITLE
Control API Architecture

### PR DESCRIPTION
**Pull request by tobru**
This PR builds on top of:

* #58

## Checklist

- [X] Try to isolate changes into separate PRs (to build a better changelog).
- [X] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/references/architecture/control-api.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
+= {controlapi} Architecture
+
+This page describes the xref:references/glossary.adoc#_control_api[{controlapi}].
+It adheres to the decision taken in xref:explanation/decisions/control-api.adoc[{controlapi}] and it follows the features asked in xref:references/functional-requirements.adoc[Functional Requirements].
```
Comment (ccremer) [inline docs/modules/ROOT/pages/references/architecture/control-api.adoc:4]:
```suggestion
It adheres to the decision taken in xref:explanation/decisions/control-api.adoc[{controlapi}] and it follows the features asked in xref:references/functional-requirements/portal.adoc[Portal Requirements].
```

File: docs/modules/ROOT/pages/references/functional-requirements.adoc [hunk 2]
```diff
@@ -0,0 +1,99 @@
+= Functional Requirements
```
Comment (ccremer) [inline docs/modules/ROOT/pages/references/functional-requirements.adoc:1]: This page already exists in `portal.adoc`.
Maybe PR needs a rebase?

File: docs/modules/ROOT/partials/nav-reference.adoc [hunk 3]
```diff
@@ -11,6 +11,13 @@
 ** xref:appuio-cloud:ROOT:references/architecture/namespace-ownership.adoc[Namespace ownership]
 ** xref:appuio-cloud:ROOT:references/architecture/metering.adoc[Metering of resource usage]
 ** xref:appuio-cloud:ROOT:references/architecture/metrics-of-interest.adoc[Metrics of interest]
+** xref:appuio-cloud:ROOT:references/architecture/control-api.adoc[Control API]
+*** xref:appuio-cloud:ROOT:references/architecture/control-api-org.adoc[Organization]
+*** xref:appuio-cloud:ROOT:references/architecture/control-api-user.adoc[User]
+*** xref:appuio-cloud:ROOT:references/architecture/control-api-team.adoc[Team]
+*** xref:appuio-cloud:ROOT:references/architecture/control-api-zone.adoc[Zone]
+
+* xref:appuio-cloud:ROOT:references/functional-requirements.adoc[Functional Requirements]
```
Comment (ccremer) [inline docs/modules/ROOT/partials/nav-reference.adoc:20]: This link is probably also outdated

**Comment by tobru** [comment]
Replaced by:

* #63

**Pull request closed without merging**